### DISCUSSION TITLE
Relentless extended access ends in one day

### DISCUSSION THREAD

**Discussion by OFark** [community/community#30370] [Copilot Conversations]
It seems every time CoPilot is triggered by VS2022 it has to open a modal popup to tell me that it expires in one day. I've dismissed it about 20 times so far this morning.

**Comment by OFark** [comment]
Even with CoPilot globally disabled it still comes up randomly to tell me.

**Reply by lupinsansei** [reply]
even after I have paid and restarted VS twice, and signed out and in again, and it still keeps coming up!

**Reply by mjefferys** [reply]
Same here, paid for a year, constantly popping up that It runs out in one day and can't get suggestions.

**Reply by bgolinvaux** [reply]
I have the same problem. I have paid the yearly fee and VS (and vscode) keep on nagging me. I have tried to uninstall/reinstall the extension to no avail. This is _extremely_ annoying, especially because the popup tends to appear when VS grabs focus , which happens all the time when Alt+Tabbing between VS and browser or other app.

A quick fix would be welcome!

**Reply by chandraawardhana** [reply]
can anyone help me here, I have made payments for one year, my funds on paypal have been deducted and so far my copilot cannot be used because my trial has run out

**Comment by Johnparke888** [comment]
I have the exact same problem.
I am using Microsoft Visual Studio Professional 2022.
I have signed out / in multiple times
I see this message in output:

A device activation code EF40-77EA has been copied to the clipboard.
Successfully authorized GitHub Copilot.

But get the message "extended access ends in one day" about once a minute.

Any help greatly appreciated.



John

**Comment by hrkrx** [comment]
Same problem, I get this message 
![image](https://user-images.githubusercontent.com/5176531/185905485-faef0614-404a-4d3e-b589-d189c3de88d8.png)

every few minutes, I even have subscribed to the paid service a month ago. 

EDIT: And no matter what is clicked it always comes back

**Reply by EvaisaDev** [reply]
IT IS SO OBNOXIOUS

**Reply by bgolinvaux** [reply]
What's funny is that the display frequency of this thing seems to wildly vary. I got it several times over 10 mins, then it kept quiet for 30 mins (thought it was fixed) but is now back with a vengeance.

**Reply by OFark** [reply]
Yeah, an hour can go past and then I walk away to get a drink and I come back to three on top of each other.

**Comment by gaetanvdberge** [comment]
Same here, I got an email this morning with the receipt that my Github Copilot subscription started for a year, but every few minutes I have to click away the popup.

**Reply by bgolinvaux** [reply]
And the damned popup gets more evil by the minute and even steals away focus from other apps!  I was in the middle of a drag & drop on the explorer when the focus suddenly switched to VS... I believe this is because I angrily cancelled it a few minutes before with ESC.

**Comment by nbarbettini** [comment]
Same issue here in VS2022 and Copilot extension v1.40.0.2. I have a monthly subscription.

I got tired of it quickly. I disabled the extension and restarted VS.

**Comment by EvaisaDev** [comment]
Same here, every 5 minutes I get a popup that says
"thank you for using github copilot, your extended access ends in one day"
I tried signing out and signing back in and it keeps happening.

**Reply by EvaisaDev** [reply]
So apparently the copilot trial period ended today and I guess it completely bugged out the extension, even for paid customers :/

**Reply by EvaisaDev** [reply]
Also it does it on both VS Code and 2022

**Reply by EvaisaDev** [reply]
latest update fixes it i hear!

**Comment by aep** [comment]
same issue on neovim. The prompt appears to be server generated and i can't figure out  a workaround because the client code is obfuscated. Teaches us all a lesson trusting Microsoft i guess.

**Comment by JimAtActiv** [comment]
Same issue here, VS 2022 v17.2.4.  Tried uninstalling GitHub CoPilot and then reinstalling, no luck still get message.  Purchased 1 year activation today.  Hopefully someone can provide some sort of work around.

**Comment by markovivan** [comment]
Can cofirm, it is continuesly occuring. VS 2022 Version 17.3.0. Attempted a fresh re-install of copilot as well.

**Comment by MikeMcLoughlin** [comment]
Same here with VS2022, it's driving me mad - I've paid too. Like other users, I've uninstalled, reinstalled, tried Copilot Settings and Dismiss, nothing works. Great product though - love it!

**Comment by Baldr-DK** [comment]
I am having the same issue, I was just billed, but it pops up constantly in my VS2022, and making it almost impossible to work.
I have several IDE's open, and it just massacres my work flow. :(

**Comment by Erik-Rogers** [comment]
Same issue in VS2022. Github CoPilot 1.40.0.2
VS2022 17.2.5

**Comment by kzu** [comment]
The latest 1.41.x version fixes this for me.

**Reply by MikeMcLoughlin** [reply]
Great, thank you

**Reply by bgolinvaux** [reply]
🙏🏻

**Comment by Erik-Rogers** [comment]
Yesterday’s update fixed the issue for me. Have you tried updating the Copilot extension?

From: Chandra Wardhana ***@***.***>
Sent: Monday, August 22, 2022 11:45 PM
To: community/community ***@***.***>
Cc: Erik Rogers ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [community/community] Relentless extended access ends in one day (Discussion #30370)

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you validate the sender and know the content is safe. Please forward this email to ***@***.******@***.***> if you believe this email is suspicious.



can anyone help me here, I have made payments for one year, my funds on paypal have been deducted and so far my copilot cannot be used because my trial has run out

**Comment by MikeMcLoughlin** [comment]
It's all okay for me with the latest version.

**Reply by bgolinvaux** [reply]
Same for me. The fix was fast :)

**Reply by Johnparke888** [reply]
Same here. VS must have done an update, because the problem went away...

**Comment by grapevinegizmos** [comment]
All fixed thanks.
